quantum/nodes/toffoli-gate/toffoli-gate.js

```diff
@@ -9,7 +9,7 @@ module.exports = function(RED) {
     RED.nodes.createNode(this, config);
     this.name = config.name;
     this.qubits = [];
-    this.targetPosition = config.targetPosition;
```

Comment:
What's the point of this change ? 
Seems incorrect to me

Reply:
Since we wouldn't want to manually change the value of the targetPosition from within the JavaScript code I thought it would be better to make it a constant. Let me know if I've broken a Node-RED convention by doing this.

Reply:
I believe it is the case for most `config.xxx` variables since they only depend on the user's HTML input.
I think we should keep the `this.xxx` as this is how the Node-RED documentation is.

If we ever decide to change it, we'll have to do the same for almost all of these `default` variables.

Reply:
Thanks for pointing that out. I've changed it back now.